src/components/censusmap.tsx

```diff
@@ -112,6 +114,15 @@ const CensusTractMap: NextPage = () => {
             scrollWheelZoom={true}
             style={{ height: "385px" }}
           >
+            <EsriLeafletGeoSearch
```

Comment:
Nitpicks: 
- Consider adding the `expanded={true}` prop so that the placeholder text will be shown on page load instead of not being shown until the input is clicked in. 
- Current placeholder text also overflows out of the input making it not clear on what should be searched. 
- In the non React version of the geosearch a magnifying glass icon is used instead of the text box on page load and once the icon is clicked the text box pops open. 
```suggestion
            <EsriLeafletGeoSearch
                 position="topleft"
                 expanded={true}
                 placeholder="Search for your address"
```
Don't mind the nits, this is a great feature. Nice work!!!